docs/configuration.md

```diff
@@ -223,7 +225,8 @@ of the most common options to set are:
     stored on disk. This should be on a fast, local disk in your system. It can also be a
     comma-separated list of multiple directories on different disks.
 
-    NOTE: In Spark 1.0 and later this will be overridden by SPARK_LOCAL_DIRS (Standalone), MESOS_SANDBOX (Mesos) or
+    <br/>
+    <em>Note:</em> In Spark 1.0 and later this will be overridden by SPARK_LOCAL_DIRS (Standalone), MESOS_SANDBOX (Mesos) or
```

Comment:
We could probably remove this or just remove "In Spark 1.0 and later"

Reply:
Nit: unrelated change.

Reply:
Fixing a nit around the code can be okay. This is the last instance in this page like:
![screen shot 2019-01-28 at 11 11 58 am](https://user-images.githubusercontent.com/6477701/51812688-942bc200-22ed-11e9-9fcd-a02227bfa567.png)

Reply:
I wouldn't reject the commit because of it, but it isn't a good practice to mix changes just because they are small.